### PR TITLE
added borrow links in custom book details and auth links in header

### LINKED ISSUE #3: add login button/form for pre-authenticaton

### PR DESCRIPTION
**Pull request by skomputer**
Closes #3 and #4.

### CODE REVIEW

File: src/server/index.tsx [hunk 1]
```diff
@@ -42,7 +42,22 @@ function handleRender(req, res) {
         );
         res.status(200).send(renderFullPage(html, state));
       }).catch(err => {
-        res.status(404).send(err);
+        // if 401, render catalog root
```
Comment (aslagle) [inline src/server/index.tsx]: wouldn't we want to render the login popup?

Comment (aslagle) [inline src/server/index.tsx]: that works, so I guess it happens on the client?

Comment (skomputer) [inline src/server/index.tsx]: yes, the client still routes the url as usual

**Comment by aslagle** [comment]
:+1:

**Pull request merged**